WORKPLAN.md

```diff
@@ -0,0 +1,36 @@
+# 2017-2018 Work Plan
+### LD4 Community Working Group on Reconciliation
+
+## About
+
+The outline of this work plan was decided at [our group kick-off meeting on 22 May 2017](https://github.com/LD4P/ld4-community-recon/wiki/2017-05-22.md). To be revisited as needed during the process of work and work calls. This skeleton work plan and any future changes to this work plan are made as pull requests with review by the whole group.
+
+## Evolving Work Plan
+
+1. [Gather/Identify Use Cases (2 months, June - July):](https://github.com/LD4P/ld4-community-recon/wiki/Entity-Resolution,-Reconciliation,-and-Lexicalization-Use-Cases-Work)
+    - Identify common use cases for reconciliation & resolution as a group;
+    - Make use cases expressed in a common way (i.e. shared template);
```

Comment:
Sure thing. I'm pitching that is should be `issue_template.md` because that is guaranteed to be what people will see when they make an issue. @arwenhutt, are you okay with that? Can we move the valuable parts of discussion on #20 into `issue_template.md`?